apps/s_time.c

```diff
@@ -403,7 +403,8 @@ static SSL *doConnection(SSL *scon, const char *host, SSL_CTX *ctx)
     /* ok, lets connect */
     for (;;) {
         i = SSL_connect(serverCon);
-        if (BIO_sock_should_retry(i)) {
+        if (SSL_get_error(serverCon, i) == SSL_ERROR_WANT_CONNECT) {
+abort();
```

Comment:
actually I begin to wonder of all the want_connect and the want_read / write stuff
really is good for anything, when there is no way ho to enable non-blocking ops.